quality_control/models/qc_test_category.py

```diff
@@ -12,12 +12,13 @@ class QcTestTemplateCategory(models.Model):
     @api.one
     @api.depends('name', 'parent_id')
     def _get_complete_name(self):
-        names = [self.name]
-        parent = self.parent_id
-        while parent:
-            names.append(parent.name)
-            parent = parent.parent_id
-        self.complete_name = " / ".join(reversed(names))
+        if self.name:
```

Comment:
Please don't make the diff bigger and increase cyclomatic complexity. Make instead:
```
if not self.name:
    return
...
```

Reply:
I think it's better to just add a or '' to the code, like this:

names = [record.name or '']

But it works, so, aproved.